package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,7 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_SIXTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
On 31 Mar, 2016, at 00:17, kain88-de wrote:

> We need a little bit more formal way to deal with deprecations and how to check that we remove functions for the targeted releases.

Do you mean like adding a test that *fails* if a function still exists in a release?

In this particular case I have rather been advocating tests that test what we want to keep https://github.com/MDAnalysis/mdanalysis/issues/703